pkg/service/repair/controller.go

```diff
@@ -177,57 +173,51 @@ func (c *rowLevelRepairController) TryBlock(hosts []string) (bool, allowance) {
 	return true, a
 }
 
+func (c *rowLevelRepairController) parallelLimit() int {
+	p := c.intensity.Parallel()
+
+	if p <= defaultParallel {
+		return c.intensity.MaxParallel()
+	}
+
+	return p
+}
+
 func (c *rowLevelRepairController) shouldBlock(hosts []string, intensity float64) bool {
 	// ALLOW if nothing is running
 	if !c.Busy() {
 		return true
 	}
 
-	// DENY if any host has max nr of jobs already running
+	// DENY if any host has a job running already. We don't allow more than a single repair job per host.
 	for _, h := range hosts {
-		if c.jobs[h] >= c.limits[h].Default {
+		if c.jobs[h] >= 1 {
+			//fmt.Printf("all jobs: %d, Too many jobs for %s (%d). Blocking.\n", c.allJobs, h, c.jobs[h])
```

Comment:
?

Reply:
I removed the last patch from the original PR that was adding these printouts.